webhooks/wh-general.md

```diff
@@ -7,7 +7,7 @@ You will receive events for all the properties you are integrated with, and can
 
 The message encapsulates all events occurring at the same time which belong to event types to which you have subscribed.
 For example, if you are interested in reservations you will subscribe to `ServiceOrderUpdated` events (a reservation is a type of Service Order)
```

Comment:
remove newline?

Reply:
Is there a newline in the middle of a sentence? Then yes, remove it.

Reply:
Removed